includes/admin/admin-pages.php

```diff
@@ -122,7 +122,7 @@ function give_is_admin_page( $passed_page = '', $passed_view = '' ) {
 	global $pagenow, $typenow;
 
 	$found          = true;
-	$get_query_args = isset( $_GET ) ? array_map( 'strtolower', $_GET ) : array();
+	$get_query_args = ! empty( $_GET ) ? @array_map( 'strtolower', $_GET ) : array();
```

Comment:
@kevinwhoffman for a quick solution I suppress error on `array_map` function because `strtolower` will throw a warning if pass value other then string and `$_GET` can contain array also. Let me know if you think that is not a good idea.

Reply:
@ravinderk Do you think it would be safer to replace `array_map()` with a foreach loop that checks if the value is a string before calling `strtolower()`? That way we would prevent any possibility of an array being passed and causing a warning.